src/modules/job-manager/urgency.c

```diff
@@ -97,7 +97,7 @@ void urgency_handle_request (flux_t *h,
     orig_urgency = job->urgency;
     if (event_job_post_pack (ctx->event, job,
                              "urgency", 0,
-                             "{ s:i s:i }",
+                             "{ s:I s:i }",
                              "userid", cred.userid,
                              "urgency", urgency) < 0)
```

Comment:
missed the typecast on this one

Reply:
Nice catch! Thanks, I've fixed that one and pushed the result.